sdcm/tester.py

```diff
@@ -123,6 +123,7 @@ def wrapper(*args, **kwargs):
         try:
             return method(*args, **kwargs)
         except Exception:
+            TEST_LOG.exception('Exception in wrapped method %s', method.__name__)
```

Comment:
If you are logging that an exception happened, you might as well log the exception type.